Gemfile.jruby-1.9.lock

```diff
@@ -94,7 +94,7 @@ GEM
     http_parser.rb (0.6.0-java)
     i18n (0.6.9)
     insist (1.0.0)
-    jar-dependencies (0.1.15)
+    jar-dependencies (0.2.1)
```

Comment:
Is only jruby-kafka, but not sure it need it at the end of the day also. I think we should experiment on removing it at all and see what happen. At less try an integration test before moving forward.